docs/Install-Guide/Configure.md

```diff
@@ -115,14 +115,12 @@ Brick3: node03.yourdomain.net:/export/sdb1/brick
 ```
 
 This shows us essentially what we just specified during the volume
-creation. The one this to mention is the `Status`. A status of `Created`
-means that the volume has been created, but hasn’t yet been started,
-which would cause any attempt to mount the volume fail.
+creation. The one key output worth noticing is `Status`.
+A status of `Created` means that the volume has been created,
+but hasn’t yet been started, which would cause any attempt to mount the volume fail.
```

Comment:
The above code and this para is not correctly formatted. You might want to recheck it.